CMakeLists.txt

```diff
@@ -347,8 +371,10 @@ if(SDL2COMPAT_TESTS)
   test_program(torturethread "test/torturethread.c")
   test_program(testgl2 "test/testgl2.c")
   test_program(testgles "test/testgles.c")
-  test_program(testgles2 "test/testgles2.c")
-  test_program(testgles2_sdf "test/testgles2_sdf.c;test/testutils.c")
+  if(HAVE_OPENGLES)
```

Comment:
This should be `HAVE_OPENGLES2`. `HAVE_OPENGLES` check itself should be added to `testgles`.

Reply:
Indeed, fix this.